templates/hooks/cache_cluster/sdk_update_post_set_output.go.tpl

```diff
@@ -0,0 +1,16 @@
+    if pendingModifications := resp.CacheCluster.PendingModifiedValues; pendingModifications != nil {
+		if pendingModifications.NumCacheNodes != nil {
+			ko.Spec.NumCacheNodes = pendingModifications.NumCacheNodes
+		}
+		if pendingModifications.CacheNodeType != nil {
+			ko.Spec.CacheNodeType = pendingModifications.CacheNodeType
+		}
+		if pendingModifications.TransitEncryptionEnabled != nil {
+			ko.Spec.TransitEncryptionEnabled = pendingModifications.TransitEncryptionEnabled
+		}
+	}
+	if err == nil {
+		// Setting resource synced condition to false will trigger a requeue of
+		// the resource. No need to return a requeue error here.
+		ackcondition.SetSynced(&resource{ko}, corev1.ConditionFalse, &condMsgCurrentlyUpdating, nil)
+	}
```

Comment:
I think that we do not need to requeue after an Update api call.

Reply:
I checked the generated code. I think we don't need to have this here.